src/Analyzers/CSharp/CodeFixes/UseExpressionBody/UseExpressionBodyCodeFixProvider.cs

```diff
@@ -44,12 +44,18 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
             var diagnostic = context.Diagnostics.First();
             var documentOptionSet = await context.Document.GetOptionsAsync(context.CancellationToken).ConfigureAwait(false);
 
+#if CODE_STYLE // 'CodeActionPriority' is not a public API, hence not supported in CodeStyle layer.
```

Comment:
@CyrusNajmabadi This was the only place that I had to make a semantic code change. I am not sure we want to expose this API to third party fixers - or at least not in the current form where they can bump up their priority to High. Maybe we can expose a public API that only allows the fixer to lower the priority? What do you think? Any case, I am going to file a tracking issue which we can discuss on Monday design meeting.